src/components/MaterialUI/Inputs/PredefinedElements/SearchControl.js

```diff
@@ -131,13 +145,8 @@ const SearchControl = ({ placeholder, defaultValue = "", searchOptions, onSearch
 		}
 	};
 
-	const onClear = event => {
-		onSearch(searchOption, "");
-		inputRef.current.value = "";
-		inputRef.current.focus();
-
-		event.preventDefault();
-		event.stopPropagation();
+	const handleChange = e => {
```

Comment:
Was this for debugging?  ;)

Reply:
this was the old code to clear the field. I used a new technique that removes the need for an onClick handler. A button of type="reset" resets the form without any extra code ! :)